src/app/routes/cpsAsset/getInitialData/index.js

```diff
@@ -128,14 +126,6 @@ export default async ({
       throw handleError('CPS asset data fetch error', status);
     }
 
-    const { mostWatched } = processMostWatched({
```

Comment:
If you haven't already, can we make sure that this function is also deleted?

Reply:
I can't find anything else left when I search 'processMostWatched', so hopefully I have